R/qenv-show.R

```diff
@@ -16,5 +16,32 @@
 #' @importFrom methods show
 #' @export
 setMethod("show", "qenv", function(object) {
-  rlang::env_print(object@.xData)
+  env <- get_env(object)
+  header <- cli::col_blue(sprintf("<environment: %s>", rlang::env_label(env)))
+  locked <- if (environmentIsLocked(env)) {
+    " [L]"
+  } else {
+    ""
+  }
+  parent <- sprintf("Parent: <environment: %s>", rlang::env_label(rlang::env_parent(env)))
+  cat(cli::style_bold(paste0(header, locked)), sep = "\n")
+  cat(parent, "\n")
+
+  shown <- ls(object)
+  lapply(shown, function(x) {
```

Comment:
We don't necessarly want to keep 1-1 match, with `rlang::env_print()` (the [L] indicator for each binding seems overkill)

I do like the `Bindings:` bolded header before the variable names.

It's self-explanatory, but it gives structure to the print. I would even consider adding it to the hidden objects.

![image](https://github.com/user-attachments/assets/30c74ec6-97d4-40fe-a4aa-e09a1e55b7a6)


```suggestion
  if (length(shown > 0L)) cat(cli::style_bold("Bindings:\n"))
  lapply(shown, function(x) {
```

Reply:
**Bindings** yes, but **Hidden Bindings:** not IMO. I thought about having section hidden: before and decided to not do so. I think dot-prefixed greyed out is a sufficient information

Reply:
Sounds good :+1: